shared/pack.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
```

Comment:
That's the script to run to generate the forceios-xxx.tgz and forcedroid-xxx.tgz package.
To generate both packages (in the current working directory)
./shared/pack.js --os=ios,android